examples/asteroids/main.py

```diff
@@ -3,6 +3,7 @@
 import operator
 
 from pygame.math import Vector2
+from pgzero.builtins import pause, is_paused
```

Comment:
You don't need to import the builtins - anything in builtins is already available.